pkg/http/api.go

```diff
@@ -135,14 +135,19 @@ func RevokeAuthToken(host string, verifyTLS bool, token string) (map[string]inte
 }
 
 // DownloadSecrets for specified project and config
-func DownloadSecrets(host string, verifyTLS bool, apiKey string, project string, config string, metadata bool) ([]byte, Error) {
+func DownloadSecrets(host string, verifyTLS bool, apiKey string, project string, config string, metadata bool, jsonFlag bool) ([]byte, Error) {
 	var params []queryParam
 	params = append(params, queryParam{Key: "project", Value: project})
 	params = append(params, queryParam{Key: "metadata", Value: strconv.FormatBool(metadata)})
 
 	headers := apiKeyHeader(apiKey)
-	headers["Accept"] = "text/plain"
-	statusCode, response, err := GetRequest(host, verifyTLS, headers, "/enclave/v1/configs/"+config+"/secrets", params)
```

Comment:
Shoot, this is actually a breaking change because the new endpoint ignores the `metadata` query param (which also means I should remove all `metadata` code). How do you want to handle this- make all our discussed changes "breaking", or temporarily support both endpoints so that v1 can get json download support?

If no one is actually asking for the json feature, I say we just do a big v2 release with all these changes. The affected user base has to be extremely small still.

Reply:
I say lets do a big v2 release. I think that is the safest bet.